src/sagas/ui.js

```diff
@@ -50,10 +53,15 @@ export function* exportProject() {
   );
 }
 
+export function* clearConsoleEntries() {
+  yield put(focusLine('console', 0, 0));
```

Comment:
In general we want to structure our Redux code so that it is reactive rather than imperative—actions should describe _what happened_, then the reducers and sagas should know what to do in response to the action. One simple rule of thumb is that we should never dispatch two different actions from the same context. As a corollary, we shouldn’t have a saga whose only job is to consume one action and dispatch another.

Here we are dispatching an action with an imperative intent: “focus this line”. But there is already an action being dispatched which says what happened: the `CLEAR_CONSOLE_ENTRIES` action. The `ui` reducer can thus update its state (specifically `requestedFocusedLine`) accordingly, with no need for a second action dispatch.

Since this particular bit of functionality is being tackled in a separate PR already, I think the best bet would be to just revert the commit that introduced this logic and we’ll merge this PR as it was previously.